vhdbuilder/packer/configure-windows-vhd.ps1

```
@@ -863,8 +874,9 @@ function Test-AzureExtensions {
     if (Test-Path "C:\Packages\Plugins") {
         $actualExtensions = (Get-ChildItem "C:\Packages\Plugins").Name
         if ($actualExtensions.Length -gt 0) {
-            Write-Log "Azure extensions are not expected. Details: $($actualExtensions | Out-String)"
-            exit 1
+            # TODO: Fix this.
```

I think that we should raise the error in production.